apps/services/auth/delegation-api/infra/delegation-api.ts

```diff
@@ -54,12 +54,20 @@ export const serviceSetup = (services: {
         prod: 'IS/GOV/5402696029/Skatturinn/ft-v1',
       },
       COMPANY_REGISTRY_REDIS_NODES: REDIS_NODE_CONFIG,
+      SYSLUMENN_HOST: {
+        dev: 'https://api.syslumenn.is/staging',
+        staging: 'https://api.syslumenn.is/staging',
+        prod: 'https://api.syslumenn.is',
+      },
+      SYSLUMENN_TIMEOUT: '10000',
```

Comment:
Staging referenced on dev again and timeout is quite high

Reply:
Fixed.